README.md

```diff
@@ -36,3 +41,8 @@ This [Circus Train](https://github.com/HotelsDotCom/circus-train) plugin enables
 The BigQuery plugin works by extracting the BigQuery table data into Google Storage using Google APIs, and then using Circus Train
 listeners to convert the BigQuery metadata into a Hive table object. The data is then replicated from source to replica using 
 the metadata from this mocked Hive table.
+
+# Legal
+This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).
+
+Copyright 2016-2018 Expedia Inc.
```

Comment:
Just 2018 here, we didn't start work on any of this code in 2016 did we? Not sure what inceptionYear in POM is but it should be 2018.